dockers/docker-teamd/Dockerfile.j2

```diff
@@ -25,7 +25,7 @@ RUN dpkg -i \
 debs/{{ deb }}{{' '}}
 {%- endfor %}
 
-COPY ["start.sh", "teamd.sh", "/usr/bin/"]
+COPY ["teamd.sh", "/usr/bin/"]
```

Comment:
Why remove start.sh here? It still seems necessary. And why add teamd.sh, as it appears you deleted the file in this PR?

Reply:
thanks was to remove the file teamd.sh